internal/coreinternal/golden/testdata/timestamp-norm/after_normalize.yaml

```diff
@@ -0,0 +1,78 @@
+resourceMetrics:
+  - resource:
+    scopeMetrics:
+      - metrics:
+          - description: Timestamp normalization with attributes
+            name: test.first
+            sum:
+              aggregationTemporality: 2
+              dataPoints:
+                - asInt: "0"
+                  attributes:
+                    - key: a
+                      value:
+                        stringValue: a
+                  startTimeUnixNano: "0"
+                  timeUnixNano: "1000000"
+                - asInt: "0"
+                  attributes:
+                    - key: b
+                      value:
+                        stringValue: b
+                  startTimeUnixNano: "0"
+                  timeUnixNano: "500000"
+                - asInt: "0"
+                  attributes:
+                    - key: a
+                      value:
+                        stringValue: a
+                  startTimeUnixNano: "0"
+                  timeUnixNano: "500000"
+                - asInt: "0"
+                  attributes:
+                    - key: d
+                      value:
+                        stringValue: d
+                  startTimeUnixNano: "0"
+                  timeUnixNano: "500000"
+                - asInt: "0"
+                  attributes:
+                    - key: d
+                      value:
+                        stringValue: d
+                  startTimeUnixNano: "0"
+                  timeUnixNano: "500000"
+                - asInt: "0"
+                  attributes:
+                    - key: a
+                      value:
+                        stringValue: a
+                  startTimeUnixNano: "0"
+                  timeUnixNano: "1500000"
+              isMonotonic: true
+            unit: By
+          - description: Timestamp normalization without attributes
+            name: test.second
+            sum:
+              aggregationTemporality: 2
+              dataPoints:
+                - asInt: "0"
+                  startTimeUnixNano: "0"
+                  timeUnixNano: "1500000"
+                - asInt: "1"
+                  startTimeUnixNano: "0"
+                  timeUnixNano: "1000000"
+                - asInt: "5"
+                  startTimeUnixNano: "0"
+                  timeUnixNano: "1000000"
+                - asInt: "10"
+                  startTimeUnixNano: "0"
+                  timeUnixNano: "500000"
+                - asInt: "50"
+                  startTimeUnixNano: "0"
+                  timeUnixNano: "2000000"
```

Comment:
Do we have any tests that apply both sorting and timestamp normalization? I can appreciate test them in isolation, but ultimately we want to see the result of both as well.

Reply:
Wouldn't the results of both end up being any of the current golden files in the change log since the sort work has been applied to the golden files and pushed in a previous PR, while the timestamps are being applied here? I can definitely make another test if you still would like one, but seeing as the timestamp normalization works independent of any sorting, the test would be purely for visual purposes.

Reply:
The purpose would be as an overall test of the golden package. The actual golden files would use the functionality, but the testing of the golden package should be done locally.